README.md

```diff
@@ -1,9 +1,37 @@
 # Null VM
 
-Verifiable VM made in collaboration with Lambdaclass and 3MI
+Verifiable VM made in collaboration with [Lambdaclass](https://lambdaclass.com/) and [3MI Labs](https://www.3milabs.tech/)
 
 Non-null name TBD
 
+We are developing an open-source verifiable virtual machine that allows users to prove the correctness of the execution of a given program with an input stream.
+
+Right now, this is a project under development and experimentation and must not be used in production!
+
+## Design choices
+
+- The Instruction Set Architecture is RISCV64IM
+- The proof system should be transparent (no trusted setup) and post-quantum secure (hash-based)
+- The security should be 128 bits of security
```

Comment:
I'm not sure if we are going to be on 128 or close, we need to double check? It's fine to start here though

```suggestion
- The security should be over 100 bits of security
```

Reply:
Also I'd say that it's provable